libenv/sysinfo.c

```diff
@@ -721,7 +721,7 @@ static void GetNameInfo3(EvalContext *ctx)
     Log(LOG_LEVEL_VERBOSE, "Additional hard class defined as: %s", CanonifyName(workbuf));
 
     snprintf(workbuf, CF_BUFSIZE, "%s_%s", VSYSNAME.sysname, VSYSNAME.release);
-    EvalContextClassPutHard(ctx, workbuf, "source=agent,derived-from=sys.sysname,derived-from=sys.release");
+    EvalContextClassPutHard(ctx, workbuf, "inventory,attribute_name=none,source=agent,derived-from=sys.sysname,derived-from=sys.release");
```

Comment:
What does `attribute_name=none` do?

Reply:
My understanding, is that `attribute_name=none` suppresses the item from being included in the inventory UI. I don't know why we use `inventory,attribute_name=none` instead of `report` which should cause it to be collected to the hub.